scripts/lint-readme.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
+const path = require('path');
+
+const packageRoot = process.cwd();
+const monorepoRoot = path.resolve(`${packageRoot}/../../..`);
```

Comment:
nit: This works currently because this is only being called in workspaces 3-levels deep: `plugins/*/*`.  A possible alternative would be:

```suggestion
const monorepoRoot = path.resolve(__dirname, '..');
```

Reply:
Thanks, I didn't know the `__dirname` trick. learned something new :)